client/src/components/Navigation.css

```diff
@@ -0,0 +1,6 @@
+.Navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default setting of flex, so you don't need it here. :)

Reply:
I was playing around with the ```justify-content``` properties.